### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by tkachuk2291**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -15,5 +15,18 @@
...
+un_copy = []
+sorted_variables = {}
+for i in range(len(list_a)):
+    if type(list_a[i]) in list_b:
+        copy.append(list_a[i])
+        sorted_variables["immutable"] = copy
+    else:
+        un_copy.append(list_a[i])
+        sorted_variables["mutable"] = un_copy
+print(sorted_variables)
```
Comment (vsmutok) [inline app/main.py:32]: this code is difficult to understand

Comment (vsmutok) [reply app/main.py:32]: bad variable names

File: app/main.py [hunk 2]
```diff
@@ -15,5 +15,18 @@
...
-# write your code here
+list_a = [lucky_number, pi, one_is_a_prime_number, name,
+          my_favourite_films, profile_info, marks, collection_of_coins]
+list_b = [int, float, complex, bool,
+          str, tuple, frozenset]
+copy = []
+un_copy = []
+sorted_variables = {}
+for i in range(len(list_a)):
+    if type(list_a[i]) in list_b:
```
Comment (vsmutok) [inline app/main.py:26]: better to use `isinstance`

File: app/main.py [hunk 3]
```diff
@@ -15,5 +15,18 @@
     "Sergio": 3,
 }
 collection_of_coins = {1, 2, 25}
-
-# write your code here
+list_a = [lucky_number, pi, one_is_a_prime_number, name,
+          my_favourite_films, profile_info, marks, collection_of_coins]
+list_b = [int, float, complex, bool,
+          str, tuple, frozenset]
```
Comment (vsmutok) [inline app/main.py:21]: it is better to write here only mutable data types, because there are fewer

File: app/main.py [hunk 4]
```diff
@@ -15,5 +15,18 @@
...
+copy = []
+un_copy = []
+sorted_variables = {}
+for i in range(len(list_a)):
+    if type(list_a[i]) in list_b:
+        copy.append(list_a[i])
+        sorted_variables["immutable"] = copy
+    else:
+        un_copy.append(list_a[i])
+        sorted_variables["mutable"] = un_copy
```
Comment (vsmutok) [inline app/main.py:31]:
```suggestion
        sorted_variables["mutable"].append(list_a[i])
```

File: app/main.py [hunk 5]
```diff
@@ -15,5 +15,18 @@
...
-
-# write your code here
+list_a = [lucky_number, pi, one_is_a_prime_number, name,
+          my_favourite_films, profile_info, marks, collection_of_coins]
+list_b = [int, float, complex, bool,
+          str, tuple, frozenset]
+copy = []
+un_copy = []
+sorted_variables = {}
+for i in range(len(list_a)):
```
Comment (vsmutok) [inline app/main.py:25]: no need to use `range`
```suggestion
for elem in(list_a):
```

**Review by vsmutok — CHANGES_REQUESTED** [review]

**Review by tkachuk2291 — COMMENTED** [review]
Edi solutions

**Pull request closed without merging**